stable/kafka-manager/values.yaml

```diff
@@ -0,0 +1,151 @@
+# ------------------------------------------------------------------------------
+# Kafka Manager:
+# ------------------------------------------------------------------------------
+
+## Service account configuration
+## Ref: https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/
+##
+serviceAccount:
+  create: true
+  ## Define serviceAccount name. Defaults to fully qualified name or "default"
+  ## when create is false
+  ##
+  name: ""
+
+## Specs for the Kafka-manager image
+##
+image:
+  repository: zenko/kafka-manager
```

Comment:
Why this image is so big (906MB) ?
sheepkiller/kafka-manager is 130MB and all you need is to update version in Dockerfile

Reply:
Fixed